sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/util/SearchCriteria.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.sandbox.team5.dao.util;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public class SearchCriteria {
+    private PageRequest pageRequest;
+
+    private SearchCriteria() {
+    }
+
+    public SearchCriteria(PageRequest pageRequest) {
+        this.pageRequest = pageRequest;
+    }
+
+    public <T, U> Page<T> find(JpaRepository<T, U> repository) {
```

Comment:
This class will be used as DTO, not as service. For example see how `org.hibernate.criterion.DetachedCriteria` is used. Then it means that we will use separate method in the common repository `find` or `search`.